index.js

```diff
@@ -25,9 +25,10 @@ async function loginHeroku() {
 async function buildPushAndDeploy() {
   const appName = core.getInput('app_name');
   const dockerFilePath = core.getInput('dockerfile');
+  const buildOptions = core.getInput('options')? core.getInput('options') : "";
```

Comment:
Two comments on this. 

1. Add space before the question mark
```suggestion
  const buildOptions = core.getInput('options') ? core.getInput('options') : "";
```

2. Minor code styling suggestion
```suggestion
  const buildOptions = core.getInput('options')
    ? core.getInput('options')
    : "";
```

Reply:
Commited :)